images/dockerregistry/config.yml

```diff
@@ -23,5 +23,6 @@ middleware:
         pullthrough: true
         enforcequota: false
         projectcachettl: 1m
+        blobrepottl: 10m
```

Comment:
`linkedblobcachettl` doesn't seem right since the ttl applies not only to local layer links but to the remote blobs as well. What about `blobrepositorycachettl`?